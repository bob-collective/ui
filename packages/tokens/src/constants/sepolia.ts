import { ChainId } from '@gobob/chains';

import { USDC, USDT, WBTC } from './common';
import { WETH9 } from './weth';

export const sepoliaTokens = {
  weth: WETH9[ChainId.SEPOLIA],
  wbtc: WBTC?.[ChainId.SEPOLIA],
  usdc: USDC?.[ChainId.SEPOLIA],
  usdt: USDT?.[ChainId.SEPOLIA]
};
