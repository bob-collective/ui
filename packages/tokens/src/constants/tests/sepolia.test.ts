import { describe, it, expect } from 'vitest';

import { sepoliaTokens } from '../sepolia';

describe('Sepolia Tokens Snapshot', () => {
  it('should match snapshot for sepoliaTokens', () => {
    expect(sepoliaTokens).toMatchSnapshot();
  });
});
