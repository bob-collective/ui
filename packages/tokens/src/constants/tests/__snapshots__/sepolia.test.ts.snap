// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Sepolia Tokens Snapshot > should match snapshot for sepoliaTokens 1`] = `
{
  "usdc": ERC20Token {
    "address": "0x1c7D4B196Cb0C7B01d743Fbc6116a902379C7238",
    "chainId": 11155111,
    "decimals": 6,
    "isNative": false,
    "isToken": true,
    "name": "USDC",
    "projectLink": undefined,
    "symbol": "USDC",
  },
  "usdt": undefined,
  "wbtc": undefined,
  "weth": ERC20Token {
    "address": "0x7b79995e5f793A07Bc00c21412e50Ecae098E7f9",
    "chainId": 11155111,
    "decimals": 18,
    "isNative": false,
    "isToken": true,
    "name": "Wrapped Ether",
    "projectLink": "https://weth.io",
    "symbol": "WETH",
  },
}
`;
